Packs/QualysVulnerabilityManagement/Integrations/QualysVulnerabilityManagement/QualysVulnerabilityManagement.py

```diff
@@ -835,6 +879,71 @@ def create_ip_list_dict(res_json: Dict[str, Any], type_of_dict: str) -> Dict[str
     return {}
 
 
+@logger
+def build_ip_and_range_dicts(ips_and_ranges: List[str]) -> List[List[Dict[str, str]]]:
+    """
+    This function separates the list of ips and ranges to two lists, one of singles ips
+    and the other of ranges of ips
+    Args:
+        ips_and_ranges: A list that might contain both ips and ranges of ips
+    Returns: A list that has one list which consists of single value dictionaries of ips
+             and another list which consists of single values dictionaries of ranges
+    """
+    list_of_ips = []
+    list_of_ranges = []
+    for value in ips_and_ranges:
+        if '-' in value:
+            list_of_ranges.append({'range': value})
+        else:
+            list_of_ips.append({'ip': value})
+    return [list_of_ips, list_of_ranges]
+
+
+@logger
+def create_single_host_list(ip_and_range_lists: Dict[str, Union[str, List]]) -> List[str]:
+    """
+    This function creates a single list containing both single ips and ranges of ips
+    Args:
+        ip_and_range_lists: A dictionary that can have either a single ip as a streing or
+        a list of single ips in the key 'Address' and/or a single range as a streing or
+        a list of range of ips in the key 'Range'
+    Returns: A list that has both ips and ranges of ips
+    """
+    ip_and_range_list = []
+
+    if 'Address' in ip_and_range_lists:
+        if isinstance(ip_and_range_lists['Address'], str):
+            ip_and_range_list.extend([ip_and_range_lists['Address']])
+        else:
+            for address in ip_and_range_lists['Address']:
+                ip_and_range_list.append(address)
+    if 'Range' in ip_and_range_lists:
+        if isinstance(ip_and_range_lists['Range'], str):
+            ip_and_range_list.extend([ip_and_range_lists['Range']])
+        else:
+            for ip_range in ip_and_range_lists['Range']:
```

Comment:
Is `ip_and_range_lists['Range']` is either string or a list? Are we sure that's a safe iteration?

Reply:
Yes, as explained in the docstring - When it's a single ip it is received as a single string, when there's more than one ip it's a list. So the iteration will work as expected.